src/index.html

```diff
@@ -10,8 +10,150 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <h1></h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="form-field">
+            Surname:
+          <input type="text" name="Surname">
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="Name">
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input 
+          type="number"
+          name="Age"
+          min="1"
+          max="100"
+          value="12"
+          >
```

Comment:
```suggestion
          <input 
            type="number"
            name="Age"
            min="1"
            max="100"
            value="12"
          >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) fix your code style all over the page